test/OpenTelemetry.Exporter.Zipkin.Tests/ZipkinExporterTests.cs

```diff
@@ -95,7 +95,9 @@ public void SuppressesInstrumentation()
     {
         const string ActivitySourceName = "zipkin.test";
         Guid requestId = Guid.NewGuid();
-        TestActivityProcessor testActivityProcessor = new TestActivityProcessor();
+#pragma warning disable CA2000 // Dispose objects before losing scope
+        TestActivityProcessor testActivityProcessor = new();
+#pragma warning restore CA2000 // Dispose objects before losing scope
```

Comment:
Is there any reason not to use a `using` here?

```suggestion
        using TestActivityProcessor testActivityProcessor = new();
```

Reply:
The scenario is pretty common for all cases.

The Test method is kind of factory for the processor/exporter and pass the ownership to the other objects.

In this case it is by

```csharp
        using var tracerProvider = Sdk.CreateTracerProviderBuilder()
            .AddSource(activitySourceName)
            .AddProcessor(testActivityProcessor)
            .AddProcessor(exportActivityProcessor) // here
            .AddHttpClientInstrumentation()
            .Build();
```

Then it is disposed by [TracerProviderSdk](https://github.com/open-telemetry/opentelemetry-dotnet/blob/643e80ef53a34c2818b6a2a458d3e9e1ab1b8f44/src/OpenTelemetry/Trace/TracerProviderSdk.cs#L383-L384)

Reply:
Ohhh. That makes sense. Thanks for explaining.